src/main/resources/db/migration/V3_26__add_new_actioned_by_table.sql

```diff
@@ -0,0 +1,121 @@
+    CREATE TABLE actioned_by
+    (
+        id                      serial          NOT NULL PRIMARY KEY,
+        booker_reference        text            NULL UNIQUE,
+        user_name               varchar(60)     NULL UNIQUE,
+        user_type               VARCHAR(80)     NOT NULL,
+        create_timestamp        timestamp       NOT NULL default current_timestamp
+    );
+
+    CREATE INDEX idx_action_by_user_type ON actioned_by(user_type);
+
+    INSERT INTO actioned_by(user_name,user_type)
+            SELECT actioned_by,user_type FROM event_audit
+                                         WHERE "type" not in ('PRISON_VISITS_BLOCKED_FOR_DATE','PRISONER_RELEASED_EVENT')
+                                         group by actioned_by,user_type;
```

Comment:
Should I bother doing PUBLIC userType here also?

Reply:
If this goes in before public booking goes live, it should be fine?

Reply:
should go live soon,...